realm/realm-library/src/androidTest/java/io/realm/TypeBasedNotificationsTests.java

```diff
@@ -1402,25 +1399,12 @@ public void onChange() {
 
     // ****************************************************************************************** //
     // UC 5.
-    // Callback should be notified if we call refresh (even without getting the REALM_CHANGE yet)
+    // Callback should be notified if we call refresh
```

Comment:
Is this still a valuable test case? Even without calling `refresh`, test won't fail i think?